src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -68,8 +68,11 @@ public override bool Execute()
                             i.SetMetadata("RelativeBlobPath", $"symbols/{fileName}");
                             return i;
                         });
-                        var packages = ItemsToPush.Where(i => !symbolItems.Contains(i));
-                        List<string> packageItems = GetPackageStringLists(packages);
+
+                        var packageItems = ItemsToPush.Where(i => !symbolItems.Contains(i)).Select(i => 
+                        {
```

Comment:
You don't really need the brackets here. I would just do `Select(i => i.ItemSpec);`